renovate.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["config:base", "schedule:earlyMondays"],
+  "extends": ["config:base", "schedule:before 3am on the first day of the month"],
```

Comment:
I would maybe just specify first day of the month here, because in my experience Renovate may not get to spin up all the PR's it can between midnight and 3am.